node/src/utils.rs

```diff
@@ -607,4 +609,10 @@ mod tests {
 
         assert_eq!(extracted, expected);
     }
+
+    #[test]
+    fn resolve_address_rejects_ipv6() {
```

Comment:
Not a blocker, but since I'm paranoid, I'd consider adding:
```
fn resolve_address_accepts_ipv4()
```

Reply:
-.-

Reply:
See 4aa04b746